src/hooks/useRouteAssetId/useRouteAssetId.ts

```diff
@@ -65,9 +64,8 @@ export const useRouteAssetId = () => {
 
   const assetId = useMemo(() => {
     const routeAssetId = getRouteAssetId(location.pathname)
-    const foxPageRouteAssetId = getFoxPageRouteAssetId(location.pathname)
 
-    return decodeURIComponent(routeAssetId ?? foxPageRouteAssetId)
+    return decodeURIComponent(routeAssetId ?? '')
```

Comment:
```suggestion
    return decodeURIComponent(routeAssetId)
```

Reply:
One or the other was previously defined (`foxPageRouteAsseId` would always be either `foxAssetId` or `foxyAssetId`), but now that the right side is removed, we need a fallback

Reply:
Oooh, I see 🙏